PI2-prensa-de-latas/software-frontend
# Login page

This PR create the profile page and its components header, feed score

## Issue 
Login page


## Description

To test this go to the:
/Start there you can see the Login and Register button that redirect to the pages
there you can see a default profile
## Screenshots (if appropriate):



## Types of changes


## Checklist:
